src/command/info/info.c

```diff
@@ -77,15 +78,22 @@ STRING_STATIC(INFO_STANZA_STATUS_MESSAGE_NO_BACKUP_STR,             "no valid ba
 #define INFO_STANZA_STATUS_CODE_MISSING_STANZA_DATA                 3
 STRING_STATIC(INFO_STANZA_STATUS_MESSAGE_MISSING_STANZA_DATA_STR,   "missing stanza data");
 
+// Locks
+VARIANT_STRDEF_STATIC(STATUS_KEY_LOCK_VAR,                          "lock");
+VARIANT_STRDEF_STATIC(STATUS_KEY_LOCK_BACKUP_VAR,                   "backup");
+VARIANT_STRDEF_STATIC(STATUS_KEY_LOCK_BACKUP_HELD,                  "held");
```

Comment:
`STATUS_KEY_LOCK_BACKUP_HELD` should be `STATUS_KEY_LOCK_BACKUP_HELD_VAR`.

I also think these three keys should be moved up with the test of the keys.

Reply:
Okay. Modified.